modules/340-monitoring-yandex-cloud/templates/yandex-cloud-metrics-exporter/deployment.yaml

```diff
@@ -0,0 +1,109 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
```

Comment:
We discussed it internally with Nikolay and decided to move all the code to the yandex cloud provider.

Reply:
done